examples/bpk-component-card/examples.js

```diff
@@ -108,12 +108,25 @@ const WithHrefDividedCardExample = () => (
   />
 );
 
+const UnElevatedDividedCardExample = () => (
+  // eslint-disable-next-line backpack/use-tokens
+  <div style={{ width: 343 }}>
+    <BpkDividedCard
+      primaryContent={longMessage}
+      secondaryContent={shortContent}
+      orientation={ORIENTATION.vertical}
+      isElevated={false}
+    />
+  </div>
+);
+
 const MixedExample = () => (
   <div>
     <DefaultExample />
     <WithoutPaddingExample />
     <DefaultDividedCardExample />
     <VerticalDividedCardExample />
+    <UnElevatedDividedCardExample />
```

Comment:
Seems you forgot to update the snapshots causing the visual tests to fail

Reply:
Thanks for your review, have update storyshots via run `npm run jest:visual-tests:update` locally.